spec/forEach_spec.js

```diff
@@ -0,0 +1,25 @@
+import { forEach } from '../src/forEach'
+
+describe('forEach', ()=> {
+  it('iterates over the collection when the input is an array and a function', () => {
+    expect(forEach ( [1,2], console.log()) ).toEqual( [1,2] )
+   })
+
+  it('iterates over the collection when the input is an object and a function', () => {
+    expect(forEach ( {a:1}, console.log()) ).toEqual( {a:1 } )
+   })
+
+  it('iterates over the collection when the input is an object and a function', () => {
+    expect(forEach ( {a:1}, console.log()) ).toEqual( {a:1 } )
+   })
+
+  it('returns null if you put null as the iteratee', () => {
+    expect(forEach ( {a:1}, null ) ).toEqual( null )
+   })
+
+  it('returns undefined if you put undefined as the iteratee', () => {
+    expect(forEach ( [1,2], undefined ) ).toEqual( [1,2] )
+   })
+
```

Comment:
We need to include tests to verify:
* Identity function is used if iteratee is not provided
* Iteratee is invoked with value, index, array for array inputs, and value, key, object for object inputs.